ovs/dal/hybrids/storagedriver.py

```diff
@@ -24,14 +24,14 @@
 from ovs.extensions.storageserver.storagedriver import StorageDriverClient
 from ovs.log.logHandler import LogHandler
 
-logger = LogHandler.get('dal', name='hybrid')
-
 
 class StorageDriver(DataObject):
     """
     The StorageDriver class represents a Storage Driver. A Storage Driver is an application
     on a Storage Router to which the vDisks connect. The Storage Driver is the gateway to the Storage Backend.
     """
+    _logger = LogHandler.get('dal', name='hybrid')
```

Comment:
I wonder whether a hybrid can use the logger available in the ```DataObject```. If not, try by - in ```DataObject``` - assign the logger to sth like ```self._logger```, and then I think you can just use ```self._logger``` in the hybrids as well.